.circleci/config.yml

```diff
@@ -195,7 +197,7 @@ commands:
           command: |
             . ${VENV_PATH}/bin/activate
             pip config set global.progress_bar off
-            pip install -U 'pip<19.0.0' wheel pip-tools
+            pip install -U 'pip==18.1' 'setuptools==40.8.0' 'wheel==0.33.6' 'pip-tools<3.9.0'
```

Comment:
Are these constraints still necessary with the cache change?

Reply:
Probably not all of them.  I'll try removing some of them (it's rather quick because the problem happens only a few minutes into the CI build.).